store/storage_manager_builder.go

```diff
@@ -500,7 +500,9 @@ func (smb *StorageManagerBuilder) buildLocalSigner(
 	}
 
 	accountID := crypto.PubkeyToAddress(signer.PrivateKey.PublicKey)
-	pendingBalance, err := ethClient.PendingBalanceAt(ctx, accountID)
```

Comment:
off topic but this pending balance doesn't do anything - we should instead be querying the `PaymentVault` directly. Updated here: https://github.com/Layr-Labs/eigenda-proxy/issues/372

Reply:
Oh snap good catch... cc @litt3

Reply:
Yeah, this check was half baked. Probably should just be removed, until we properly consider what tools proxy needs to make payments easier for users